test/fixtures/es-modules/package-cjs-named-error/json-hack/fail.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  comeOn: 'fhqwhgads'
```

Comment:
everybody to the limit

Reply:
Maybe `foo` would be more appropriate here?

Reply:
i don't see why anything could possibly be more appropriate than this particular reference